generators/entity/index.js

```diff
@@ -925,6 +932,33 @@ module.exports = EntityGenerator.extend({
             },
             {
                 when: function (response) {
+                    return response.relationshipAdd === true &&
+                        response.relationshipType === 'many-to-one';
```

Comment:
@deepu105  I am adding required field to relationships selected as many to one , many to many and one to one. that's enough right?  many to one I don't think its required. Right?